web3.md

```diff
@@ -1692,16 +1714,36 @@ Gas Limit Call
     syntax KItem ::= "#firefly_setGasLimit"
  // ---------------------------------------
     rule <k> #firefly_setGasLimit => #rpcResponseSuccess(true) ... </k>
-         <params> [ GLIMIT:String, .JSONs ] </params>
-         <gasLimit> _ => #parseWord( GLIMIT ) </gasLimit>
+         <params>          [ GLIMIT:String, .JSONs ] </params>
+         <gasLimit>        _ => #parseWord( GLIMIT ) </gasLimit>
+         <defaultGasLimit> _ => #parseWord( GLIMIT ) </defaultGasLimit>
 
     rule <k> #firefly_setGasLimit => #rpcResponseSuccess(true) ... </k>
-         <params> [ GLIMIT:Int, .JSONs ] </params>
-         <gasLimit> _ => GLIMIT </gasLimit>
+         <params>          [ GLIMIT:Int, .JSONs ] </params>
+         <gasLimit>        _ => GLIMIT            </gasLimit>
+         <defaultGasLimit> _ => GLIMIT            </defaultGasLimit>
 
     rule <k> #firefly_setGasLimit => #rpcResponseError(-32000, "firefly_setGasLimit requires exactly 1 argument") ... </k> [owise]
 ```
 
+Gas Price Call
+--------------
+
+```k
+    syntax KItem ::= "#firefly_setGasPrice"
+ // ---------------------------------------
+    rule <k> #firefly_setGasPrice => #rpcResponseSuccess(true) ... </k>
+         <params>          [ GPRICE:String, .JSONs ] </params>
+         <gasPrice>        _ => #parseWord( GPRICE ) </gasPrice>
+         <defaultGasPrice> _ => #parseWord( GPRICE ) </defaultGasPrice>
```

Comment:
Same here, we can have one rule for converting the `String` to an `Int`, and another for loading it into the state.